policyengine_us/variables/gov/states/ky/tax/income/ky_filing_separately.py

```diff
@@ -0,0 +1,20 @@
+from policyengine_us.model_api import *
+
+
+class ky_files_separately(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Married couple file separately on the Kentucky tax return"
+    definition_period = YEAR
+    reference = "https://revenue.ky.gov/Forms/740%20Packet%20Instructions%205-9-23.pdf#page=11"
+    defined_for = StateCode.KY
+
+    def formula(tax_unit, period, parameters):
+        taxable_income_indiv = add(
```

Comment:
should this compare tax liabilities instead of taxable income? this came up in a recent issue in another state @PavelMakarchuk

Reply:
Yes, after speaking with Martin, we split it at `ky_income_tax_before_non_refundable_credits`